util/cmds/cmd_map.go

```diff
@@ -0,0 +1,84 @@
+package cmds
+
+// RedisCallback function type
+type DiceDBCallback func(interface{}) interface{}
+
+// command2callback map
+var command2callback = map[string]DiceDBCallback{
+	"AUTH":         renderSimpleString,
```

Comment:
Can we make this other way around? Like group methods to commands list?
It'd reduce redundancy.
Something like below and add a finder on top of this
```
// Create hashsets (maps) for each command group
var simpleStringCommands = map[string]struct{}{
	"AUTH":    {}, "PING":    {}, "SELECT":  {},
	"SET":     {}, "RESTORE": {}, "FLUSHDB": {},
}
```